src/features/components/templates/footer/download.tsx

```diff
@@ -9,27 +9,32 @@ import clsx from 'clsx'
 import { StaticImage } from 'gatsby-plugin-image'
 import { CustomLink } from '@deriv-com/components'
 import { localize } from 'components/localization'
+import useRegion from 'components/hooks/use-region'
 
 const sharedClasses =
     'flex items-center justify-center gap-gap-md p-general-sm rounded-[4px] border-solid border-xs border-opacity-black-100'
 
 const DownloadBadges = () => {
+    const { is_appgallery_supported, is_ios_supported, is_appgallery_and_ios_supported } =
+        useRegion()
     return (
         <div className="flex flex-col gap-gap-md max-lg:flex-1">
-            <CustomLink
-                href="https://appgallery.huawei.com/app/C103801913"
-                target="_blank"
-                className={clsx(sharedClasses, 'order-last')}
-            >
-                <LabelPairedHuaweiAppGalleryLgIcon fill="#000000b8" />
-                <StaticImage
-                    src="../../../../images/common/migration/footer/explore-appgallery.png"
-                    alt={localize('_t_explore it on appgallery_t_')}
-                    formats={['webp', 'auto']}
-                    width={90}
-                    placeholder="none"
-                />
-            </CustomLink>
+            {is_appgallery_supported && is_appgallery_and_ios_supported && (
+                <CustomLink
+                    href="https://appgallery.huawei.com/app/C103801913"
```

Comment:
dowe need to put this in const file?

Reply:
@mayuran-deriv  @Nuzhy-Deriv can confirm on if there was a specific reason they didn’t put it in the const ,as it should be same link we use for the banner in the deriv go page

Reply:
no specific reason @mitra-deriv. If we are having these urls in a const file, we can use from there.